Tests/Runtime/UnityVectorExtensionTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿using UnityEngine;
+using UnityEngine.TestTools;
+using NUnit.Framework;
+using Cinemachine.Utility;
+
+public class RuntimeDampingTest
+{
+    [Test]
+	public void FindIntersectionTests()
+    {
+        {
+            var l1_p1 = new Vector2(0, 1);
+            var l1_p2 = new Vector2(0, -1);
+            var l2_p1 = new Vector2(-1, 0);
+            var l2_p2 = new Vector2(1, 0);
+            UnityVectorExtensions.FindIntersection(l1_p1, l1_p2, l2_p1, l2_p2, out bool linesIntersect,
+                out bool segmentsIntersect, out Vector2 intersection);
+            Assert.IsTrue(linesIntersect);
+            Assert.IsTrue(segmentsIntersect);
+            Assert.IsTrue(Mathf.Abs(intersection.x) < 1e-8f &&
+                          Mathf.Abs(intersection.y) < 1e-8f); // intersection should be Vector2.zero
+        }
+        {
+            var l1_p1 = new Vector2(0, 1);
+            var l1_p2 = new Vector2(0, 0);
+            var l2_p1 = new Vector2(-1, 0);
+            var l2_p2 = new Vector2(1, 0);
+            UnityVectorExtensions.FindIntersection(l1_p1, l1_p2, l2_p1, l2_p2, out bool linesIntersect, 
+                out bool segmentsIntersect, out Vector2 intersection);
+            Assert.IsTrue(linesIntersect);
+            Assert.IsTrue(segmentsIntersect);
+            Assert.IsTrue(Mathf.Abs(intersection.x) < 1e-8f &&
+                          Mathf.Abs(intersection.y) < 1e-8f); // intersection should be Vector2.zero
+        }
+        {
+            var l1_p1 = new Vector2(0, 2);
+            var l1_p2 = new Vector2(0, 1);
+            var l2_p1 = new Vector2(-1, 0);
+            var l2_p2 = new Vector2(1, 0);
+            UnityVectorExtensions.FindIntersection(l1_p1, l1_p2, l2_p1, l2_p2, out bool linesIntersect, 
+                out bool segmentsIntersect, out Vector2 intersection);
+            Assert.IsTrue(linesIntersect);
+            Assert.IsFalse(segmentsIntersect);
+            Assert.IsTrue(Mathf.Abs(intersection.x) < 1e-8f &&
+                          Mathf.Abs(intersection.y) < 1e-8f); // intersection should be Vector2.zero
+        }
+        {
+            var l1_p1 = new Vector2(0, 2);
+            var l1_p2 = new Vector2(0, 1);
+            var l2_p1 = new Vector2(1, 2);
+            var l2_p2 = new Vector2(1, 1);
+            UnityVectorExtensions.FindIntersection(l1_p1, l1_p2, l2_p1, l2_p2, out bool linesIntersect, 
+                out bool segmentsIntersect, out Vector2 intersection);
+            Assert.IsFalse(linesIntersect);
+            Assert.IsFalse(segmentsIntersect);
+        }
+    }
+
+    [Test]
+     public void TestAngle()
+    {
+        {
+            Vector3 v1 = Vector3.up;
+            Vector2 v2 = Vector3.left;
+            float angle = UnityVectorExtensions.Angle(v1, v2);
+            Assert.AreApproximatelyEqual(angle, 90f);
```

Comment:
Assert.AreApproximatelyEqual creates compile errors for me with your test project

Reply:
It's ok for me. Maybe it was because of the class name. If it comes up again, I'll change it to Assert.IsTrue